### PR TITLE
Blog post for sprint 130

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: _includes/partials/_series-of-posts-about-request-redesign.md [hunk 1]
```diff
@@ -1 +1,4 @@
-_We started the [redesign of the request workflow](/2022/08/15/request-workflow-redesign) in August 2022. Then, in September 2022, we focused on the [support of multi-action submit requests](/2022/09/14/request-workflow-redesign-round-two) to continue now with [improvements regarding the Build Results tab and superseded conversations](/2022/10/24/request-workflow-redesign)._
\ No newline at end of file
+_We started the [redesign of the request workflow](/2022/08/15/request-workflow-redesign) in August 2022.
+Then, in September 2022, we focused on the [support of multi-action submit requests](/2022/09/14/request-workflow-redesign-round-two).
+We continued in October 2022 with [improvements regarding the Build Results tab and superseded conversations](/2022/10/24/request-workflow-redesign), and
+now we present [more improvements to Build Results and a pinch of salt](/2022/11/21/request-workflow-redesign)._
```
Comment (danidoni) [inline _includes/partials/_series-of-posts-about-request-redesign.md:4]: The title is not in sync with the one below

Comment (eduardoj) [reply _includes/partials/_series-of-posts-about-request-redesign.md]: Now it is synced.

File: _posts/development/2022-11-25-request-workflow-redesign.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Request Page Redesign - Build Results with a Pinch of Salt
+category: development
+---
+
+Once again, we worked on the request workflow.
+This time we have delivered a renovated interface to show the Build results, ready to be consumed by the most demanding palates.
+Besides that, we have cooked a couple of additions arranged on several tabs for the hungriest.
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/development/2022-11-25-request-workflow-redesign.md]: Applied. Thanks!

File: _posts/development/2022-11-28-request-workflow-redesign.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Request Page Redesign - Build Results with a Pinch of Salt
+category: development
+---
+
+Once again, we worked on the request workflow.
+This time we have delivered a renovated interface to show the Build results, ready to be consumed by the most demanding palates.
```
Comment (dmarcoux) [inline _posts/development/2022-11-28-request-workflow-redesign.md:8]: _Build_ isn't a proper noun, so it should not be capitalized:

```suggestion
This time we have delivered a renovated interface to show the build results, ready to be consumed by the most demanding palates.
```

File: _includes/partials/_series-of-posts-about-request-redesign.md [hunk 4]
```diff
@@ -1 +1,4 @@
-_We started the [redesign of the request workflow](/2022/08/15/request-workflow-redesign) in August 2022. Then, in September 2022, we focused on the [support of multi-action submit requests](/2022/09/14/request-workflow-redesign-round-two) to continue now with [improvements regarding the Build Results tab and superseded conversations](/2022/10/24/request-workflow-redesign)._
\ No newline at end of file
+_We started the [redesign of the request workflow](/2022/08/15/request-workflow-redesign) in August 2022.
+Then, in September 2022, we focused on the [support of multi-action submit requests](/2022/09/14/request-workflow-redesign-round-two).
+We continued in October 2022 with [improvements regarding the Build Results tab and superseded conversations](/2022/10/24/request-workflow-redesign), and
+now we present [Build Results with a Pinch of Salt](/2022/11/28/request-workflow-redesign)._
```
Comment (dmarcoux) [inline _includes/partials/_series-of-posts-about-request-redesign.md:4]: The following words aren't proper nouns and we're also not referring to something like the _Build Results_ tab entity, so they should not be capitalized.

```suggestion
now we present [build results with a pinch of salt](/2022/11/28/request-workflow-redesign)._
```

File: _posts/development/2022-11-28-request-workflow-redesign.md (hunk 3 above)
Comment (eduardoj) [reply _posts/development/2022-11-28-request-workflow-redesign.md]: Suggestion applied.

File: _includes/partials/_series-of-posts-about-request-redesign.md (hunk 4 above)
Comment (eduardoj) [reply _includes/partials/_series-of-posts-about-request-redesign.md]: Suggestion applied.

**Review by dmarcoux — APPROVED** [review]
Awesome work!!!

**Pull request merged**